task/buildah-oci-ta/0.4/buildah-oci-ta.yaml

```diff
@@ -799,31 +1360,51 @@ spec:
           exit 1
           ;;
         esac
-
         echo "Running syft on the source directory"
         syft dir:"/var/workdir/$SOURCE_CODE_DIR/$CONTEXT" --output "$syft_sbom_type"="/var/workdir/sbom-source.json"
         echo "Running syft on the image"
         syft oci-dir:"$(cat /shared/container_path)" --output "$syft_sbom_type"="/var/workdir/sbom-image.json"
-
         echo "[$(date --utc -Ins)] End sbom-syft-generate"
     - name: prepare-sboms
-      image: quay.io/konflux-ci/mobster:0.5.0-1752493098@sha256:51109021a96e12ad19de9f5ab7a076683250af18d98532385a033916882cd294
+      image: quay.io/konflux-ci/sbom-utility-scripts@sha256:1b714c468641e910f37c15aa53b867f0b9550a06650e07ffc0583338b963e7af
```

Comment:
It seems like a rebase went incorrect here, but this change reverts the Mobster SBOM generator to the legacy solution.